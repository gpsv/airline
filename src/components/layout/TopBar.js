import React from "react";
import { NavLink } from "react-router-dom";

export const TopBar = () => {
  return (
    <>
      <nav className="navigation">
        <NavLink
          exact="true"
          className="app-navlink "
          to="/"
        >
          Home
        </NavLink>
        <NavLink
          exact="true"
          className="app-navlink"
          to="/flightBooking"
        >
          Mis reservas
        </NavLink>
      </nav>
    </>
  );
};
