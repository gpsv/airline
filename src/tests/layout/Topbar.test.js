import { shallow } from "enzyme";
import { TopBar } from "../../components/layout/TopBar";

describe('Pruebas en <TopBar/>', () => {
  test('Debe de mostrar el componente correctamente', () => {
    const wrapper = shallow( <TopBar />);
    expect( wrapper ).toMatchSnapshot();
  })
  
});