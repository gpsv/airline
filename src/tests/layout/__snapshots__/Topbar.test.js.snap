// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <TopBar/> Debe de mostrar el componente correctamente 1`] = `
<Fragment>
  <nav
    className="navigation"
  >
    <NavLink
      className="app-navlink "
      exact="true"
      to="/"
    >
      Home
    </NavLink>
    <NavLink
      className="app-navlink"
      exact="true"
      to="/flightBooking"
    >
      Mis reservas
    </NavLink>
  </nav>
</Fragment>
`;
